app/src/main/java/org/p2p/wallet/user/interactor/UserTokensInteractor.kt

```diff
@@ -68,4 +73,11 @@ class UserTokensInteractor(
     suspend fun getUserBalance(): BigDecimal {
         return userTokensLocalRepository.getUserBalance()
     }
+
+    suspend fun setTokenHidden(mintAddress: String, visibility: String) =
```

Comment:
i think there should be a separate interactor like HiddenTokenInteractor for this logic

Reply:
maybe ) let's do this in next )